src/mscorlib/src/System/String.cs

```diff
@@ -219,13 +218,19 @@ public sealed class String : IComparable, ICloneable, IConvertible, IEnumerable
             if (count == 0) {
                 return String.Empty;
             }
-            
+
+            if (count == 1) {
+                return value[startIndex] ?? String.Empty;
```

Comment:
`return value[startIndex] == null ? string.Empty : value[startIndex];` isn't strictly equivalent due to the second indexing. To be really equivalent would need:

```C#
string ret = value[startIndex];
return ret == null ? string.Empty : ret;
```
Which begins to feel less consistent anyway.